webapp/src/app/shared/table/table.component.ts

```diff
@@ -36,10 +38,15 @@ export interface FilterItem {
     styleUrls: ['./table.component.css'],
 })
 export class TableComponent implements OnInit, AfterViewInit, OnChanges, OnDestroy {
+    @Input() idColumn;
+    @Input() selectedRowId;
     @Input() columnsAndFiltersToExcludeFromCasing = [];
     @Input() centeredColumns: { [key: string]: boolean } = {};
     @Input() columnsSortFunctionMap;
     @Input() filterFunctionMap;
+    @Input() dateCategoryList;
+    screenWidth: number;
+    whitelistColumnsSubscription: any;
```

Comment:
@sidharthjain-zemoso please, use `Subscription` instead of `any` or better, use `takeUntil(this.destroy$)` approach